config/config.go

```diff
@@ -84,6 +84,9 @@ type Log struct {
 	SlowThreshold      uint   `toml:"slow-threshold" json:"slow-threshold"`
 	ExpensiveThreshold uint   `toml:"expensive-threshold" json:"expensive-threshold"`
 	QueryLogMaxLen     uint   `toml:"query-log-max-len" json:"query-log-max-len"`
+
+	// DDLSlowOprThreshold is the threshold of ddl slow operations.
+	DDLSlowOprThreshold uint `toml:"ddl-slow-opr-threshold" json:"ddl-slow-opr-threshold"`
```

Comment:
I think if we make this configurable, it's better to use HTTP API, it doesn't require a restart of the server.

Reply:
Then the config option can be removed.